### PR TITLE
feat: show the Button status for the include robot extension

### PR DESCRIPTION
**Pull request by dido18**
The connection button status was shared among the `Arduino Include robot` and the `BLE Arduino Sense` extensions. 

Add the button to the `Arduino include robot`  extension.

image

NOTE: the shared `DEVICE` has been removed.

### CODE REVIEW

File: scratch-arduino-extensions/scratch-vm/src/extensions/scratch3_arduino/index.js [hunk 1]
```diff
@@ -88,6 +85,8 @@ class IncludeRobot {
      * @type {Runtime}
      */
     this._runtime = runtime;
+
+    this._device = new Peripheral(this._runtime, IncludeRobot.EXTENSION_ID);
```
Comment (lucarin91) [inline scratch-arduino-extensions/scratch-vm/src/extensions/scratch3_arduino/index.js:89]: I will keep the same device singleton at least, in this way we can still reuse the same connection for both extension

Comment (dido18) [reply scratch-arduino-extensions/scratch-vm/src/extensions/scratch3_arduino/index.js:89]: I hoped but unfortunately we can't.
The  `Peripheral` needs the ID of the extension to "register" the popup to the button.

Comment (lucarin91) [reply scratch-arduino-extensions/scratch-vm/src/extensions/scratch3_arduino/index.js:89]: I don't think that this is an issue, you can still reuse the connection.

Comment (dido18) [reply scratch-arduino-extensions/scratch-vm/src/extensions/scratch3_arduino/index.js:89]: What is the objective to maintain a single BLE connection? 
If it is related to be able to manage both the extensions, the same result could be obtained even with two different connections. This is possible by connecting both extensions.

This is an example:
 - I connected both the extensions to the robot
 - I managed to see the data coming from the BLE Sense and to send the command.

image

**Review by lucarin91 — APPROVED** [review]

**Pull request merged**